lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -110,7 +110,7 @@ private function build($data) {
       'id' => isset($northstar_user) ? $northstar_user->id : $data->field_northstar_id_value,
       'first_name' => dosomething_helpers_extract_field_data($drupal_user->field_first_name),
       'last_initial' => $last_initial,
-      'photo' => dosomething_helpers_extract_field_data($drupal_user->photo),
+      'photo' => isset($drupal_user->picture) ? $drupal_user->picture : NULL,
```

Comment:
Yep, hence sometime in the near-ish future, rather than immediately and without warning. 😉 Once Phoenix has Northstar IDs in it though, that part of the proxy can be removed.